src/helpers/pivot/pivot_helpers.ts

```diff
@@ -230,7 +230,7 @@ export function toNormalizedPivotValue(
 
 function normalizeDateTime(value: CellValue, granularity: Granularity) {
   if (!granularity) {
-    throw new Error("Missing granularity");
+    granularity = "month";
```

Comment:
slightly shorter and easier with `pivotTimeAdapter(granularity ?? "month")` instead of reassigning the argument